### ISSUE TITLE
Job Parameters are overwritten

### ISSUE THREAD

**Issue by pamelasarkisyan** [jenkinsci/pipeline-github-plugin#120]
### Jenkins and plugins versions report

As part of adding parameters for the job all existing parameters are getting removed and only the new ones are visible.

Lines that are probably introducing this behaviour are:

https://github.com/jenkinsci/pipeline-github-plugin/blob/master/src/main/java/org/jenkinsci/plugins/pipeline/github/trigger/GitHubEventSubscriber.java#L132-L134

Environment: 
[Jenkins 2.361.3](https://www.jenkins.io/)
Pipeline: GitHub -> Version2.8-141.7e985db66f30

### What Operating System are you using (both controller, and any agents involved in the problem)?

N/A

### Reproduction steps

Create a job with any parameters.
Trigger the job via a comment. 
Navigate to Parameters for the Job that started. 

### Expected Results

You should see the existing parameters of the Job with the default values as defined in the job. (In my case it is declarative job so those are defined in Jenkinsfile)

### Actual Results

The only parameters that you will see are: `GITHUB_COMMENT` and `GITHUB_COMMENT_AUTHOR`
Existing parameters have been removed.

### Anything else?

https://github.com/jenkinsci/pipeline-github-plugin/commit/ebc7956cb980c7897c8e951fcde01cf4dd6a7731
This commit is aiming to add environment variables (commit name is: expose comments as env variables) but it instead add parameters.

**Comment by aaronwalker** [comment]
@pamelasarkisyan when you trigger the job via a comment your expectation is you'd have the default values for any custom  parameters correct? Since there is no way to trigger the custom using a comment and also pass parameters to the job.

**Comment by pamelasarkisyan** [comment]
@aaronwalker Yes exactly I want to start the job via comment and keep the custom parameters with their default values.

Can GITHUB_COMMENT and GITHUB_COMMENT_AUTHOR be added as environment variables instead of parameters?

**Comment by aaronwalker** [comment]
@pamelasarkisyan ok let me take a look

**Comment by preetam810** [comment]
This is a major blocker for those who use parameters in their pipelines. It would be great to have this fixed please.
